entrypoint.sh

```diff
@@ -110,4 +124,5 @@ fi
 sudo -u node -- sh -c "\
 RDSINSTANCES=${instances} SECURITYGROUP=${sg} WEBHOOK_URL_PARAMETER=${webhookparameter} \
 ALERT_USERNAME=${username} ALERT_CHANNEL=${channel} \
-cdk deploy --profile ${profile}"
\ No newline at end of file
+RDS_PREFERRED_NAMES=${rds_preferred_names} CPU_THRESHOLD=${cpu_alert_threshold} \
```

Comment:
I think it would be good to guard against an invalid number of preferred names here before deployment.
Something like: ```[ `echo $RDSINSTANCES | sed 's/,/\n/g' | wc -l` -eq `echo $RDS_PREFERRED_NAMES | sed 's/,/\n/g' | wc -l` ]``` (But nicer, aha)

I guarantee this will save someone a huge headache at some point.

The guard should still be in place within the Lambda to protect at runtime.